rr_gazebo/CMakeLists.txt

```diff
@@ -10,6 +10,11 @@ find_package( catkin REQUIRED COMPONENTS
         rr_platform
 )
 
+# Fix Protobuf include if needed
+if(NOT Protobuf_INCLUDE_DIR)
+	set(Protobuf_INCLUDE_DIR /usr/include/)
```

Comment:
This could just be an "Evan's laptop" problem. This added chunk gets the code to compile on some computers, but it might also be what's breaking the CI build